tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,227 @@
+@api
+@replication @replication-aditions
+Feature: Contentpool media replication basically works.
+
+  Make sure that media referenced in content can be pulled from remote contentpool server.
+
+  @replication-aditions-test-media
+  Scenario: Creating media entities
+    Given I am logged in to contentpool
+
+    # Add media entitiy
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 1"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 1"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 2"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 2"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 3"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 3"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Image media 4"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+    Then I save media to array with title "BEHAT: Image media 4"
+
+  @javascript @replication-aditions-default
+  Scenario: Replication media works
+    # Create article with teaser
+    Given I am logged in to contentpool
+    When I visit path "node/add/article" on contentpool
+    And I fill in "title[0][value]" with "BEHAT: Media"
+    And I fill in "field_seo_title[0][value]" with "test media"
+    And I select "-Cooking" from "edit-field-channel"
+    And I press "field_teaser_media_entity_browser_entity_browser"
+    Then I wait for AJAX to finish
+    Then I wait for ".views-row:nth(0)" in entity browser "image_browser"
```

Comment:
Please modify selector to use title/name/alt text so it's clear what is selected